mcpwm_foc.c

```diff
@@ -2793,7 +2795,7 @@ void mcpwm_foc_adc_int_handler(void *p, uint32_t flags) {
 
 		// Set motor phase
 		{
-			if (!motor_now->m_phase_override) {
+			if (!motor_now->m_phase_override ) {
```

Comment:
Get rid of the whitespace. The end result should be this line isn't changed at all.